brayns/parameters/RenderingParameters.cpp

```diff
@@ -45,28 +45,30 @@ const std::string PARAM_DETECTION_ON_DIFFERENT_MATERIAL =
 const std::string PARAM_DETECTION_NEAR_COLOR = "detection-near-color";
 const std::string PARAM_DETECTION_FAR_COLOR = "detection-far-color";
 const std::string PARAM_EPSILON = "epsilon";
-const std::string PARAM_CAMERA_TYPE = "camera-type";
+const std::string PARAM_CAMERA = "camera";
 const std::string PARAM_HEAD_LIGHT = "head-light";
 const std::string PARAM_VARIANCE_THRESHOLD = "variance-threshold";
 
 const std::string ENGINES[2] = {"ospray", "optix"};
-const std::string RENDERERS[7] = {"basic",
+const std::string RENDERERS[7] = {"default",
                                   "proximity",
                                   "simulation",
                                   "particle",
                                   "geometrynormals",
                                   "shadingnormals",
                                   "scientificvisualization"};
-const std::string RENDERER_NAMES[7] = {"basic",
-                                       "proximityrenderer",
-                                       "simulationrenderer",
-                                       "particlerenderer",
-                                       "raycast_Ng",
-                                       "raycast_Ns",
-                                       "scivis"};
 
-const std::string CAMERA_TYPES[5] = {"perspective", "stereo", "orthographic",
-                                     "panoramic", "clipped"};
+const std::string RENDERER_INTERNAL_NAMES[7] = {"basic",
+                                                "proximityrenderer",
+                                                "simulationrenderer",
+                                                "particlerenderer",
+                                                "raycast_Ng",
+                                                "raycast_Ns",
+                                                "scivis"};
+
+const std::string CAMERA_TYPE_NAMES[5] = {"perspective", "stereo",
```

Comment:
the JSON name also has to change to default, no? for both, camera and renderer.

Reply:
It's already done in the jsonSerialization.h file. CAMERA_TYPE_NAMES defines the internal 'ospray' name of the camera, not the one that is exposed in JSON.